pkg/testing/integration/program.go

```diff
@@ -480,16 +480,6 @@ func (pt *programTester) getPipenvBin() (string, error) {
 	return getCmdBin(&pt.pipenvBin, "pipenv", pt.opts.PipenvBin)
 }
 
-// getPythonVersion returns the requested Python version to use when running this test.
-// Defaults to 3.6 if not specified.
-func (pt *programTester) getPythonVersion() string {
-	if pt.opts.PythonVersion != "" {
```

Comment:
Should we also delete the `PythonVersion` member from from `ProgramTestOptions`?

Reply:
yep, will do.